sdk/communication/azure-communication-phonenumbers/azure/communication/phonenumbers/_phone_numbers_client.py

```diff
@@ -95,8 +95,10 @@ def begin_purchase_phone_numbers(
             for LRO operations if no Retry-After header is present.
         :rtype: ~azure.core.polling.LROPoller[None]
         """
+        polling_interval = kwargs.pop('polling_interval', 2)
```

Comment:
First of all, kudos for using the default value for the pop function, second this '2' is a magic number that would be more readable within a constant like `_DEFAULT_POLLING_INTERVAL_IN_SECONDS = 2` of course change seconds by the real unit of the pooling interval.